### DISCUSSION TITLE
Turn instructions are displayed although "Off" has been defined

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#683] [Q&A] [answered]
Although "Off" has been defined in /Settings/Routing/Appearance/Turn instructions (zoom), the turn points are displayed after leaving the route, e.g. through a RoadBlock. The dots disappear again by stopping and restarting the navigation.
<img width="200" src="https://github.com/user-attachments/assets/1c00f2a0-45af-4de2-8116-9d70a34a846a"/>. . .<img width="200" src="https://github.com/user-attachments/assets/026d2efd-871c-4b65-9884-1d652f1e4efe"/>
Of course, it doesn't bother me too much.

**Comment by devemux86** [comment]
@WolfgangZ42 Thanks for the report.

> the turn points are displayed after leaving the route, e.g. through a RoadBlock.

What do you mean "leaving the route"? Did the off route warning icon appear at the top?
The 2nd image does not show the off route warning icon at the top. You are still on route?

**Reply by WolfgangZ42** [reply]
On route again. (like "on road again ..." 😉)

**Reply by devemux86** [reply]
@WolfgangZ42 Can you explain the steps with more details?

When exactly the dots disappear / appear?

**Reply by WolfgangZ42** [reply]
- Set "Off" in /Settings/Routing/Appearance/Turn instructions (zoom)
- Define any destination and start navigation from your current location
   -> no turn points shown
- initiate a roadblock let say for 100m (cause a recalculation)
   -> turn points appear
- stop navigation and start it again
   -> turn points disappear

**Reply by devemux86** [reply]
@WolfgangZ42 Thanks for the extra details.

So it happens after a rerouting, now I understand and can reproduce it.
It will work properly in the next update.

**Comment by devemux86** [comment, marked as answer]
- Fixed in https://github.com/devemux86/cruiser/discussions/684
